meta-lmp-base/recipes-containers/docker-compose/docker-compose/cli-config-support-default-system-config.patch

```diff
@@ -1,4 +1,4 @@
-From 6385df5e8c5c4f3eeff0962ce7a182e9aac68f2c Mon Sep 17 00:00:00 2001
```

Comment:
Same.

Reply:
have that on https://github.com/foundriesio/meta-lmp/pull/1444. fixed now